test/performance/config/performance-reporter.ts

```diff
@@ -108,6 +110,7 @@ export function curateResults(
 		listViewOpen: average( results.listViewOpen ),
 		minListViewOpen: minimum( results.listViewOpen ),
 		maxListViewOpen: maximum( results.listViewOpen ),
+		navigate: median( results.navigate ),
```

Comment:
Looks like we need to decide here which calculation method to go with (median vs average). Can be dealt with in a follow-up PR I guess!

Reply:
yes, we have a mix right now which is not idea. Also, it's not great that we have a performance reporter that has to refer to the keys of all the metrics from all the jobs. Ideally each job has its own metrics and the reporter is a bit agnostic.

Reply:
> Ideally each job has its own metrics and the reporter is a bit agnostic.

This should be pretty straightforward at this point. We just need to decide whether we want to apply the average or the median as that would still need to be a part of the reported I guess. Not sure what to do with the min* and max* metrics though - should we just mark them to be calculated as such in the tests?

Reply:
The jobs can also produce the metrics directly (they can choose if they want average, median, min, max...) on their own no?

Regardless, I think the min/max metrics could be removed because at this point, I feel any metric that is not on codevitals is useless.

Reply:
> The jobs can also produce the metrics directly (they can choose if they want average, median, min, max...) on their own no?

Ah, yes. We can make each metric configurable rather than impose the calc method.

> Regardless, I think the min/max metrics could be removed because at this point, I feel any metric that is not on codevitals is useless.

Sounds good!